lib/resources/postgres.rb

```diff
@@ -81,6 +60,32 @@ def verify_dirs
       end
     end
 
+    def version_from_psql
+      return unless inspec.command('psql').exist?
+      inspec.command("psql --version | awk '{ print $NF }' | awk -F. '{ print $1\".\"$2 }'").stdout.strip
+    end
+
+    def verify_data_dir_location(ver = @version)
```

Comment:
See comment above re: keeping this helper method isolated rather than having it modify instance variables. That will make it easier to write tests and will remain consistent with the other functions in this resource, such as `version_from_psql`